src/System.Management.Automation/engine/InitialSessionState.cs

```diff
@@ -5339,12 +5339,17 @@ internal static SessionStateAliasEntry[] BuiltInAliases
 }
 
 # Respect PAGER, use more on Windows, and use less on Linux
-if (Test-Path env:PAGER) {
-    $moreCommand = (Get-Command -CommandType Application $env:PAGER | Select-Object -First 1).Definition
-} elseif ($IsWindows) {
-    $moreCommand = (Get-Command -CommandType Application more | Select-Object -First 1).Definition
-} else {
-    $moreCommand = (Get-Command -CommandType Application less | Select-Object -First 1).Definition
+Try {
+  if (Test-Path env:PAGER) {
+      $moreCommand = (Get-Command -CommandType Application $env:PAGER -ErrorAction Stop | Select-Object -First 1).Definition
```

Comment:
I would avoid an exception both for performance reasons (just a general guideline, probably doesn't matter here) and because this needlessly adds an error to `$Error`.

With `Select-Object -First 1`, maybe it's fine to just add `,cat` to each of the cases, e.g.

```powershell
$moreCommand = (Get-Command -CommandType Application more,cat | Select-Object -First 1).Definition
```